### ISSUE TITLE
Better README is needed

### ISSUE THREAD

**Issue by AlexDaniel** [bread-maker/firmware#1]
Currently there's no info regarding supported microcontrollers/boards, no pinout, etc. I think some minimal documentation is required.

**Comment by ClusterM** [comment]
Yes, i need to write good documentation. But i have not time for it now. I hope that i'll do it soon.

**Comment by markh-de** [comment]
Can you please at least tell us which bread making machine it is compatible with?
Just disassembled (repaired) three of these: https://www.amazon.de/AFK-BM-2N-Brotbackautomat/dp/B00007EMAE the other day, and I found the board design quite modular. So I thought about replacing the controller PCB by my own design.
